custom_components/foxess_modbus/entities/remote_control_description.py

```diff
@@ -67,6 +67,21 @@
             ),
             models=Inv.KH_119,
         ),
+        RemoteControlAddressSpec(
+            holding=ModbusRemoteControlAddressConfig(
+                remote_enable=44000,
+                timeout_set=44001,
+                active_power=[44002],
+                ac_power_limit_down=None,
```

Comment:
@TonyM1958 Are you sure this isn't mapped as 44008? It would be odd to have pwr_limit_bat_up but not this.